### PR TITLE
Serve jujugui wsgi app in tornado.

### PR DESCRIPTION
**Pull request by jaycee**

### CODE REVIEW

**Comment by jaycee** [comment]
QA:

Deploy the charm whether as a local charm or via make deploy; ensure that you see the wsgi app served on the machine that's brought up--note that it will still be broken b/c you cannot login.

File: hooks/backend.py [hunk 1]
```diff
@@ -140,7 +140,12 @@ def install(self, backend):
                 config.get('ssl-key-contents'))
 
     def start(self, backend):
-        pass
+        config = backend.config
+        utils.start_builtin_server(
+            config['ssl-cert-path'], config['serve-tests'],
+            config['sandbox'], config['builtin-server-logging'],
+            not config['secure'], config['charmworld-url'],
+            port=config.get('port'))
```
Comment (bac) [inline hooks/backend.py]: Why specify port but not the others?

File: server/guiserver/apps.py [hunk 2]
```diff
@@ -88,13 +86,15 @@ def server():
         'start_time': int(time.time()),
     }
     server_handlers.extend([
```
Comment (bac) [inline server/guiserver/apps.py]: Please combine this with the one at line 96.  No reason to keep them separate.

**Comment by bac** [comment]
Mind running 'make lint' and cleaning it up?

**Comment by bac** [comment]
Code :+1:  -- thanks.

I will do QA shortly once my tests are finished.

**Comment by bac** [comment]
QA-OK -- I see the app and the 'Connecting to Juju Env' with a spinner. I assume that is what is expected.

:+1:

File: hooks/backend.py (hunk 1 above)
Comment (jaycee) [inline hooks/backend.py]: I'm not sure what you mean--I'm just bringing back all of the available juju config options for the builtin server.

File: server/guiserver/apps.py (hunk 2 above)
Comment (jaycee) [inline server/guiserver/apps.py]: Done.

**Pull request merged**